website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -89,25 +135,27 @@ metrics:
 | name        | A unique identifier for the metric                          | new_customers                   | yes       |
-| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
+| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.3">yes (no for `derived` metrics)</VersionBlock><VersionBlock lastVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
 | label       | A short for name / label for the metric                     | New Customers                   | no        |
 | description | Long form, human-readable description for the metric        | The number of customers who.... | no        |
-| type        | The type of calculation to perform when evaluating a metric | count_distinct                  | yes       |
-| sql         | The expression to aggregate/calculate over                  | user_id                         | yes       |
+| <VersionBlock firstVersion="1.3">calculation_method</VersionBlock><VersionBlock lastVersion="1.1">type </VersionBlock> | <VersionBlock firstVersion="1.3">The method of calculation (aggregation or derived) that is applied to the expression</VersionBlock><VersionBlock lastVersion="1.1">The type of calculation to perform when evaluating a metric</VersionBlock>  | count_distinct | yes       |
+| <VersionBlock firstVersion="1.3">expression</VersionBlock><VersionBlock lastVersion="1.1">sql</VersionBlock> | The expression to aggregate/calculate over | user_id | yes       |
 | timestamp   | The time-based component of the metric                      | signup_date                     | yes       |
 | time_grains | One or more "grains" at which the metric can be evaluated   | [day, week, month]              | yes       |
 | dimensions  | A list of dimensions to group or filter the metric by       | [plan, country]                 | no        |
 | filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
 | meta        | Arbitrary key/value store                                   | {team: Finance}                 | no        |
-
+|<VersionBlock firstVersion="1.3"> window </VersionBlock> <VersionBlock lastVersion="1.2">Not yet available - added in v1.3 </VersionBlock>     | <VersionBlock firstVersion="1.3"> Used for rolling/cumulative metrics where the calculation method is applied across a range of time  </VersionBlock> <VersionBlock lastVersion="1.2">Not yet available — added in v1.3</VersionBlock>  | <VersionBlock firstVersion="1.3"> 14 days  </VersionBlock> <VersionBlock lastVersion="1.2">Not yet available — added in v1.3</VersionBlock>   | no |
 
 ### Available types
```

Comment:
can we add contextual info on what types are and what they do?

Reply:
These correspond to the type/calculation method property of the metric, where it is defined in the above table